var/spack/repos/builtin/packages/py-nvidia-dali/package.py

```diff
@@ -16,6 +16,8 @@ class PyNvidiaDali(PythonPackage):
     homepage = "https://developer.nvidia.com/dali"
     url = "https://developer.download.nvidia.com/compute/redist/"
 
+    skip_version_audit = ["platform=darwin"]
```

Comment:
I wasn't aware of this feature.  Nice.

Reply:
@alalazo added it for CUDA

Reply:
Then I probably heard about it during a meeting but this is the first time I've seen it in use.